pkg/venafi/tpp/connector_test.go

```diff
@@ -546,6 +548,71 @@ func TestRequestCertificateWithValidityDuration(t *testing.T) {
 	DoRequestCertificateWithValidityDuration(t, tpp)
 }
 
+func TestRequestCertificateWithContactEmails(t *testing.T) {
+	t.Skipf("Skipping test because the TPP instance doesn't have LDAP configured")
```

Comment:
I had to skip this test since https://supertreat.venqa.venafi.com/ only has local identities.

Do you think we could turn on the AD DS service on the VM that runs  https://supertreat.venqa.venafi.com/? @marcos-albornoz @luispresuelVenafi

Reply:
Let's leave this off, only modify the message to say something like "skipping by default because this test only works over TPP instances which have LDAP users configured."

Reply:
Thanks, I have updated the message to say that.